src/minimizer_mapper.hpp

```diff
@@ -94,6 +94,9 @@ class MinimizerMapper : public AlignerClient {
     /// of total score
     double minimizer_score_fraction = 0.9;
 
+    /// Take at maximum number of unique non-overlapping minimizers
+    size_t max_unique_min = 1000;
```

Comment:
```suggestion
    size_t max_unique_min = 500;
```
We probably should set this to match the command-line default.